services/coincap/coincap-priceusd.service.js

```diff
@@ -0,0 +1,43 @@
+import Joi from 'joi'
+import BaseCoincapService from './coincap-base.js'
+
+const schema = Joi.object({
+  data: Joi.object({
+    priceUsd: Joi.string().required(),
```

Comment:
Again, if we're going to manipulate this as a number in `priceFormat()` we should validate more strictly here. Any old string won't do.